<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deletecus</class>
 <widget class="QDialog" name="deletecus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>372</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>757</width>
    <height>372</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>757</width>
    <height>372</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>删除顾客信息</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 26pt &quot;方正舒体&quot;;</string>
       </property>
       <property name="text">
        <string>删除顾客信息</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QComboBox" name="comboBox">
           <item>
            <property name="text">
             <string>请选择删除方式</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>按单号删除</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>按编号删除</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>按姓名删除</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
